educator/src/Dscp/Educator/Web/Auth.hs

```diff
@@ -0,0 +1,72 @@
+{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE TypeOperators #-}
+-- | Nessesary types and implementation for web server authenthication
```

Comment:
Necessary. Also it's subjective, but the comment usually sticks to the `module` line like this:
```
{-# LANGUAGE Pragma1 #-}

-- | Comment
module Dscp.mem where ...
```